### ISSUE TITLE
If outrigger.key is an empty directory, skip further key handling

### ISSUE THREAD

**Issue by grayside** [phase2/docker-build#7]
An empty directory indicates the local key did not exist, if so let's treat that as though there is no key and not spew a number of confusing errors.

<details><summary>Errors from Broken outrigger.key volume mount</summary>

```bash
cat: /root/.ssh/outrigger.key: Is a directory
cp: missing destination file operand after '/root/.ssh/outrigger.key'
Try 'cp --help' for more information.
chown: missing operand after 'root:root'
Try 'chown --help' for more information.
chmod: missing operand after '600'
Try 'chmod --help' for more information.
# bitbucket.org SSH-2.0-conker_1.0.287-a1d21a7 app-128
# bitbucket.org SSH-2.0-conker_1.0.287-a1d21a7 app-126
no hostkey alg
# github.com SSH-2.0-libssh-0.7.0
# github.com SSH-2.0-libssh-0.7.0
no hostkey alg
```

</details>

**Comment by febbraro** [comment]
Is this handled as part of the Jenkins low security fix?

**Comment by febbraro** [comment]
Obviously in a different repo, but should be ported here?

**Comment by grayside** [comment]
Yeah, it's basically the same mechanism managed in each repo for simplicity. I dropped the fix for this specific issue as a minor tweak in that PR while working on the main point.

**Comment by grayside** [comment]
Fixed by #25
